test/test_cuda.py

```diff
@@ -823,20 +823,6 @@ def test_cat_autogpu(self):
         z = torch.cat([x, y], 0)
         self.assertEqual(z.get_device(), x.get_device())
 
-    def test_bernoulli(self):
-        AbstractTestCases._TestTorchMixin._test_bernoulli(self, torch.float32, torch.float64, 'cuda')
-        AbstractTestCases._TestTorchMixin._test_bernoulli(self, torch.float32, torch.float16, 'cuda')
-        AbstractTestCases._TestTorchMixin._test_bernoulli(self, torch.float16, torch.float64, 'cuda')
```

Comment:
Delete these since it have been covered by the test in `test_torch.py`, which is going to annotate with `device` and `dtype` parameters for full situation test.